WIP-CHANGELOG.md

```diff
@@ -16,6 +16,10 @@
 
 - Moved `.monitoring.rook` and `.rookCeph.enabled` keys under `rookCeph.monitoring` and `rookCeph.gatekeeperPsp`
 - The value for `networkPolicies.global.externalLoadBalancer` and `networkPolicies.global.ingressUsingHostNetwork` is now set by the init script, based on the `.global.ck8sCloudProvider` value
+- Change Trivy Operator Dashboard to only count image states once per image instead for each namespace and resource
+- Rework thanos networkpolicies with generator chart
+- Rework monitoring networkpolicies with generator chart
+- New fluentd-elasticsearch custom image tag `v4.3.9-ck8s1` which uses a new plugin for dot replacement, as this functionality has been removed from the `kubernetes_metadata_filter` plugin.
```

Comment:
Remind me, why do we need to build a custom image?
How do I find what the source for the image?
I can't see any updates to our [fork](https://github.com/elastisys/fluentd-elasticsearch)

Reply:
From what I was able to gather the main reason now is that the Gemfile for the image [upstream](https://github.com/monotek/fluentd-elasticsearch/blob/master/Gemfile) doesn't include some of the plugins that we use:

`fluent-plugin-record-modifier` 
`fluent-plugin-dedot_filter` - Added by me

I just created a [tag](https://github.com/elastisys/fluentd-elasticsearch/tree/v4.3.9-ck8s1) on our fork where you should see the config used for the image i built, e.g. the [Gemfile](https://github.com/elastisys/fluentd-elasticsearch/blob/v4.3.9-ck8s1/Gemfile).